xarray/conventions.py

```diff
@@ -786,6 +822,16 @@ def decode_cf_variable(var, concat_characters=True, mask_and_scale=True,
             dimensions = dimensions[:-1]
             data = CharToStringArray(data)
 
+    pop_to(attributes, encoding, '_Unsigned')
+    is_unsigned = encoding.get('_Unsigned', False)
+    if (is_unsigned) and (mask_and_scale is True):
```

Comment:
You don't need the extra parentheses here, and use implicit boolean checks instead of `is True`. So this should be: `if is_unsigned and mask_and_scale`